src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
```

Comment:
- [ ] Kindly delete this file since it is of no use to you instead of commenting on the codes in it. It would make your codebase look clean and organized. 👍